x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DatafeedJobsIT.java

```diff
@@ -309,7 +311,7 @@ public void testStopAndRestartCompositeDatafeed() throws Exception {
             .setMapping("time", "type=date")
             .get();
         long numDocs = randomIntBetween(32, 2048);
-        long now = System.currentTimeMillis();
+        long now = Intervals.alignToFloor(System.currentTimeMillis(), TimeValue.timeValueHours(1).millis()) + 1;
```

Comment:
this is still causing a failure :(((. Trying to figure out how to make this deterministic.

Reply:
Now fixed. Making sure all timestamps are not on the edge of buckets given the start/finish/ and interval.